lib/config/maxSockets.ts

```diff
@@ -0,0 +1,19 @@
+import * as http from 'http';
+
+/**
+ * The maximum socket configuration defaults to 50.
+ */
+const maxSocketsNumber = Number(process.env.MAX_SOCKETS) || 50;
+
+/**
+ * The default configuration for the maxSocket profile.
+ * The goal is to enforce a `maxSockets` property to properly
+ * handle load.
+ */
+const maxSocketsConfiguration: http.AgentOptions  = {
+    keepAlive: true,
```

Comment:
It's a bit strange to see `keepAlive` in a config named `maxSocketsConfiguration`, could it be named like `agentConfiguration`?

Reply:
And maybe rename the file as well

Reply:
Thank you, I did it in https://github.com/scality/httpagent/pull/6/commits/eb40166fd867aef06827b6e6efcc51419cf2e550, and I used the name you suggested. I also updated the name of the exported variable.